### ISSUE TITLE
[website] - "Discover more from ChromieCraft" Mail Shelter

### ISSUE THREAD

**Issue by TheSCREWEDSoftware** [chromiecraft/chromiecraft#7413]
### Bug Title

Mail Shelter Pop-up wrong colours.

### Current Behaviour

Some pages that have "Discover More from ChromieCraft" within the page. ("News") area.

Go to https://www.chromiecraft.com/en/create-twinks-with-carboncopy/ and scroll down.

You will have a pop-up when you react close to the start of the "Discover More" (if the page is small, if it's big it will load half way into the page). If it's intended I'm not sure. but if it's i'm pretty sure it needs a different colour for the text.

<details>
<summary>How It Looks (click me)</summary>

![image](https://github.com/user-attachments/assets/a4190046-df9f-4cbb-8bdf-dddfccf55a47)
</details>

<details>
<summary>How it Looks Selected (click me)</summary>

![image](https://github.com/user-attachments/assets/b993c001-4bd1-4efe-96c3-c3c1bf616244)
</details>

### Expected Behaviour

I would pressume that the shelter would always appear at the start of 1/4 or 1/2 of the way of reading an article. If it was big enough, otherwise shouldn't pop-up (and the bottom one being the main focused).

### Steps to Reproduce

1 - Open a Private / Incoginot Page
2 - Go to either of the links belows
3 - Scroll normally and you get the message.

This was tested on Chrome, Firefox and Edge. Where Firefox and Edge have no extentions.

### Browser

Chrome

### Device

Desktop

### Operating System

Windows 10 x64

### Extra Information

I think has to do with no tags associted, can't be sure, don't know the critea of the e-mail shelter.

### URL

https://www.chromiecraft.com/en/create-twinks-with-carboncopy/ 

https://www.chromiecraft.com/en/chromiecraft-merch-store-now-live/

https://www.chromiecraft.com/en/top-contributors-of-january-2024/

**Comment by Helias** [comment]
I had a look into the text color, I have to investigate properly it
For the discover behavior I ping @Yehonal

**Comment by Yehonal** [comment]
Yes I noticed that too, I think we can fix that with a css override in our theme. Thanks for reporting

**Comment by Yehonal** [comment]
I think this is fixed, can you please confirm

**Comment by TheSCREWEDSoftware** [comment]
Tested again with the same browsers in private / iconginto windows and still feels that it appears very earlier into the post.
